CMakeLists.txt

```diff
@@ -48,7 +48,7 @@ option(USE_CLANG_TOOLCHAIN "Build the kernel with the experimental Clang toolcha
 include(Meta/CMake/wasm_spec_tests.cmake)
 
 add_custom_target(run
-    COMMAND ${CMAKE_COMMAND} -E env "SERENITY_ARCH=${SERENITY_ARCH}" ${CMAKE_SOURCE_DIR}/Meta/run.sh
+    COMMAND "${CMAKE_COMMAND}" -E env "SERENITY_ARCH=${SERENITY_ARCH}" "${SerenityOS_SOURCE_DIR}/Meta/run.sh"
```

Comment:
nitpick: Why is the casing for `SerenityOS_SOURCE_DIR` special, shouldn't it match the casing for every other cmake variable and just be `SERENITYOS_SOURCE_DIR`?

Reply:
It's following the <ProjectName>_SOURCE_DIR convention.
https://cmake.org/cmake/help/latest/variable/PROJECT-NAME_SOURCE_DIR.html#variable:%3CPROJECT-NAME%3E_SOURCE_DIR

This one has ProjectName in the case that it was set in the call to `project()`. 

Rationale there is there's an option to Move the "top level" `CMakeLists.txt` file for Serenity into Meta/Serenity or some such, and have the "SuperBuild" be the top level. If we go with that route, then this pattern will make things easier. I made this change on the previous PR that's open, here.

https://github.com/SerenityOS/serenity/pull/9281#discussion_r684844975